diff --git a/dist/node/index.mjs b/dist/node/index.mjs
index a4cc76316326b3c5a60e00416321355f5782797e..edac4c159b3bb0990f1cd19eaf9d38a4807a38b2 100644
--- a/dist/node/index.mjs
+++ b/dist/node/index.mjs
@@ -4,6 +4,7 @@ import { convert } from "html-to-text";
 import { Suspense } from "react";
 import { Writable } from "node:stream";
 import { jsx } from "react/jsx-runtime";
+import reactDOMServer from "react-dom/server";
 
 //#region src/shared/utils/pretty.ts
 function recursivelyMapDoc(doc, callback) {
@@ -125,7 +126,6 @@ const readStream = async (stream) => {
 //#region src/node/render.tsx
 const render = async (node, options) => {
 	const suspendedElement = /* @__PURE__ */ jsx(Suspense, { children: node });
-	const reactDOMServer = await import("react-dom/server").then((m) => m.default);
 	let html$1;
 	if (Object.hasOwn(reactDOMServer, "renderToReadableStream") && typeof WritableStream !== "undefined") html$1 = await readStream(await reactDOMServer.renderToReadableStream(suspendedElement, { progressiveChunkSize: Number.POSITIVE_INFINITY }));
 	else await new Promise((resolve, reject) => {
